astroquery/ipac/ned/core.py

```diff
@@ -495,10 +500,25 @@ def extract_image_urls(self, html_in):
         html_in : str
             source from which the urls are to be extracted
 
+        format : str, optional
+            Format of spectra to return. Defaults to 'fits'.
+            Other options available: 'author-ascii', 'NED-ascii', 'VO-table'.
+
         """
         base_url = 'http://ned.ipac.caltech.edu'
+
+        extensions = {'fits': 'fits.gz',
+                      'author-ascii': 'txt',
+                      'NED-ascii': '_NED.txt',
+                      'VO-table': '_votable.xml'}
+
+        names = {'fits': 'FITS',
+                 'author-ascii': 'Author-ASCII',
+                 'NED-ascii': 'NED-ASCII',
+                 'VO-table': 'VOTable'}
+
         pattern = re.compile(
-            r'<a\s+href\s*?="?\s*?(.+?fits.gz)"?\s*?>\s*?(?:Retrieve|FITS)</a>',
+            f'<a\s+href\s*?="?\s*?(.+?{extensions[file_format]})"?\s*?>\s*?(?:Retrieve|{names[file_format]})</a>',
```

Comment:
this is the meat of the change, so I want to note explicitly: `r'\s'` and `'\s'` are the same thing, so the removal of the leading `r` is OK